app/composables/masto/masto.ts

```diff
@@ -25,14 +25,19 @@ export function mastoLogin(masto: ElkMasto, user: Pick<UserLogin, 'server' | 'to
   const url = `https://${server}`
   const instance: ElkInstance = reactive(getInstanceCache(server) || { uri: server, accountDomain: server })
   const accessToken = user.token
+  const streamingApiUrl = instance?.configuration?.urls?.streaming
 
-  const createStreamingClient = (streamingApiUrl: string | undefined) => {
-    return streamingApiUrl ? createStreamingAPIClient({ streamingApiUrl, accessToken, implementation: globalThis.WebSocket }) : undefined
-  }
+  let createStreamingClient: (streamingApiUrl: string | undefined) => mastodon.streaming.Client | undefined
 
-  const streamingApiUrl = instance?.configuration?.urls?.streaming
   masto.client.value = createRestAPIClient({ url, accessToken })
-  masto.streamingClient.value = createStreamingClient(streamingApiUrl)
+
+  if (currentUser.value !== undefined) {
+    createStreamingClient = (streamingApiUrl: string | undefined) => {
+      return streamingApiUrl ? createStreamingAPIClient({ streamingApiUrl, accessToken, implementation: globalThis.WebSocket }) : undefined
+    }
+
+    masto.streamingClient.value = createStreamingClient(streamingApiUrl)
+  }
```

Comment:
I added a check for `currentUser` before creating the streaming client to prevent the errors we get when elk.zone tries to subscribe to the web socket without a valid token.